Analysis/src/QwMollerADC_Channel.cc

```diff
@@ -362,14 +362,28 @@ void QwMollerADC_Channel::SetEventData(Double_t* block, UInt_t sequencenumber)
   return;
 }
 
+__attribute__((no_sanitize("signed-integer-overflow")))
 void QwMollerADC_Channel::SetRawEventData(){
   fNumberOfSamples = fNumberOfSamples_map;
   fHardwareBlockSum_raw = 0;
 //  Double_t hwsum_test = 0.0;
 //  std::cout <<  "*******In QwMollerADC_Channel::SetRawEventData for channel:\t" << this->GetElementName() << std::endl;
   for (Int_t i = 0; i < fBlocksPerEvent; i++) 
     {
-     fBlock_raw[i] = Int_t((fBlock[i] / fCalibrationFactor + fPedestal) * fNumberOfSamples / (fBlocksPerEvent * 1.0));
+     Float_t block_raw = (fBlock[i] / fCalibrationFactor + fPedestal) * fNumberOfSamples / (fBlocksPerEvent * 1.0);
```

Comment:
Bad idea. Float cannot obtain the necessary lsb at the clamp value. Probably was bad already but now the badness is manifest. Needs double. Unless we are in with block as shorter than 32 but then that's not manifest.

Reply:
```suggestion
     Double_t block_raw = (fBlock[i] / fCalibrationFactor + fPedestal) * fNumberOfSamples / (fBlocksPerEvent * 1.0);
```